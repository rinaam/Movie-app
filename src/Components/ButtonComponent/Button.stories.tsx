import React from "react";
import { Story, Meta } from "@storybook/react";

import ButtonComponent, {
  IButtonProps,
} from "./ButtonComponent";
import RandomIcon from '../../Assets/Icons/RandomIcon.svg';
import CloseIcon from '../../Assets/Icons/CloseIcon.svg';

export default {
  title: "Button",
  component: ButtonComponent,
} as Meta;

const Template: Story<IButtonProps> = (args) => <ButtonComponent {...args} />;

export const Primary = Template.bind({});
Primary.args = {
  variant: "primary",
  children: "Roll",
};

export const Round = Template.bind({});
Round.args = {
  variant: "round",
  children: "Load",
};

export const Floating = Template.bind({});
Floating.args = {
  variant: "floating",
  children: <RandomIcon />,
};

export const Close = Template.bind({});
Close.args = {
  variant: "close",
  children: <CloseIcon />,
};
