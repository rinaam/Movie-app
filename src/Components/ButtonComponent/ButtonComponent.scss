@use "sass:map";
@import "../../Styles/variables.scss";
@import "../../Styles/breakpoints.scss";

.btn {
  font-weight: map.get($font-weight, "s");
  border: 0;
  cursor: pointer;
  display: inline-block;
  line-height: 1;
  padding: map.get($spacing, "s");
  outline: none;
  &-primary {
    color: $white;
    height: 35px;
    background-color: $black;
  }
  &-round {
    width: 60px;
    height: 60px;
    border-radius: 100%;
    background-color: $white-smoke;
    margin: map.get($spacing, "s");
  }
  &-floating {
    @extend .btn-round;
    position: fixed;
    right: 0;
    bottom: 0;
    margin: map.get($spacing, "m");
    color: $white;
    background-color: $black;
    font-size: map.get($font-size, "m");
    box-shadow: map.get($box-shadow, "primary");
    @include breakpoint(smallDevices) {
      margin: map.get($spacing, "xl");
    }
  }
  &-close {
    background-color: $white;
  }
}