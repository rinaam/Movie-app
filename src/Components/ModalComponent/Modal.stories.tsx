import React from "react";
import { Story, Meta } from "@storybook/react";

import ModalComponent, { IModalProps } from "./ModalComponent";

export default {
  title: "Modal",
  component: ModalComponent,
} as Meta;

const Template: Story<IModalProps> = (args) => <ModalComponent {...args} />;

export const Primary = Template.bind({});
Primary.args = {
  title: "Movie Roulette",
  open: true,
  children: (
    <div>
      <p>test</p>
      <p>llllll</p>
    </div>
  ),
};
