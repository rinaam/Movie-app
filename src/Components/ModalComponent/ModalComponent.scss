@use "sass:map";
@import "../../Styles/breakpoints.scss";
@import "../../Styles/variables.scss";

.modal {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background-color: $black-opacity;
  .container {
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    background-color: $white;
    width: 100%;
    height: 100%;
    padding: map.get($spacing, "s");
    display: flex;
    flex-direction: column;
    box-sizing: border-box;
    @include breakpoint(smallDevices) {
      width: 300px;
      height: 300px;
    }
    &-header {
      display: flex;
      justify-content: space-between;
      align-items: center;
      border-bottom: 1px solid $light-grey;
    }
    &-content {
      margin: map.get($spacing, "xs");
      height: 100%;
      flex: 1;
    }
  }
}
