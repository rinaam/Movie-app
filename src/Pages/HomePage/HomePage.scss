.btn-wrapper {
  display: flex;
  justify-content: center;
}
.mdl-container {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  height: 100%;
}
