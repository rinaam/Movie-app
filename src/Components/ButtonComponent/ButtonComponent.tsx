import React from 'react';
import './ButtonComponent.scss';

export interface IButtonProps {
    variant: 'primary' | 'round' | 'floating' | 'close';
    children: JSX.Element | string;
    onClick: () => void;
}

const ButtonComponent: React.FC<IButtonProps> = ({
  variant,
  children,
}) => (
  <button
    type="button"
    className={`btn btn-${variant}`}
  >
    {children}
  </button>
);

export default ButtonComponent;
