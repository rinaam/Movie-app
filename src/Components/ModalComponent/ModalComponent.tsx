import React from "react";
import "./ModalComponent.scss";
import CloseIcon from "../../Assets/Icons/CloseIcon.svg";
import ButtonComponent from "../ButtonComponent/ButtonComponent";

export interface IModalProps {
  title: string;
  children: JSX.Element;
  open: boolean;
  onClose: () => void;
}

const ModalComponent: React.FC<IModalProps> = ({
  title,
  children,
  open,
  onClose,
}) => (
  <>
    {Boolean(open) && (
      <div className="modal">
        <div className="container">
          <div className="container-header">
            <p>{title}</p>
            <ButtonComponent variant="close" onClick={onClose}>
              <CloseIcon />
            </ButtonComponent>
          </div>
          <div className="container-content">{children}</div>
        </div>
      </div>
    )}
  </>
);

export default ModalComponent;
