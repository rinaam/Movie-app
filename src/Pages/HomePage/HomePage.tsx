import React, { useState } from "react";

import ContainerComponent from "../../Components/ContainerComponent/ContainerComponent";
import ButtonComponent from "../../Components/ButtonComponent/ButtonComponent";
import ModalComponent from "../../Components/ModalComponent/ModalComponent";
import RadioGroupComponent from "../../Components/RadioGroupComponent/RadioGroupComponent";
import RandomIcon from "../../Assets/Icons/RandomIcon.svg";
import MoviesList from "../../Containers/MoviesList/MoviesList";
import "./HomePage.scss";

export interface IMovies {
  page: number;
  results: TMovie[];
  total_pages: number;
  total_results: number;
}

export type TMovie = {
  adult?: boolean;
  backdrop_path: string;
  genre_ids: number[];
  id: number;
  original_language: string;
  original_title?: string;
  overview: string;
  poster_path: string;
  release_date?: string;
  title?: string;
  video?: boolean;
  vote_average: number;
  vote_count: number;
  popularity: number;
  first_air_date?: string;
  name?: string;
  origin_country?: string[];
  original_name?: string;
};

export interface IGenre {
  genres: TGenre[];
}

type TGenre = {
  id: number;
  name: string;
};

export const genresMock: IGenre = {
  genres: [
    {
      id: 287,
      name: "Action",
    },
    {
      id: 35,
      name: "Comedy",
    },
    {
      id: 53,
      name: "Drama",
    },
  ],
};

export const moviesMock: IMovies = {
  page: 1,
  results: [
    {
      adult: false,
      backdrop_path: "/bOGkgRGdhrBYJSLpXaxhXVstddV.jpg",
      genre_ids: [28, 12, 14, 878],
      id: 299536,
      original_language: "en",
      original_title: "Avengers: Infinity War",
      overview:
        "As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain.",
      poster_path: "/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg",
      release_date: "2018-04-25",
      title: "Avengers: Infinity War",
      video: false,
      vote_average: 8.3,
      vote_count: 6937,
      popularity: 358.799,
    },
    {
      adult: false,
      backdrop_path: "/3P52oz9HPQWxcwHOwxtyrVV1LKi.jpg",
      genre_ids: [28, 35, 878],
      id: 383498,
      original_language: "en",
      original_title: "Deadpool 2",
      overview:
        "Wisecracking mercenary Deadpool battles the evil and powerful Cable and other bad guys to save a boy's life.",
      poster_path: "/to0spRl1CMDvyUbOnbb4fTk3VAd.jpg",
      release_date: "2018-05-15",
      title: "Deadpool 2",
      video: false,
      vote_average: 7.6,
      vote_count: 3938,
      popularity: 223.011,
    },
    {
      adult: false,
      backdrop_path: "/22cUd4Yg5euCxIwWzXrL4m4otkU.jpg",
      genre_ids: [28, 878, 53],
      id: 500664,
      original_language: "en",
      original_title: "Upgrade",
      overview:
        "A brutal mugging leaves Grey Trace paralyzed in the hospital and his beloved wife dead. A billionaire inventor soon offers Trace a cure — an artificial intelligence implant called STEM that will enhance his body. Now able to walk, Grey finds that he also has superhuman strength and agility — skills he uses to seek revenge against the thugs who destroyed his life.",
      poster_path: "/adOzdWS35KAo21r9R4BuFCkLer6.jpg",
      release_date: "2018-06-01",
      title: "Upgrade",
      video: false,
      vote_average: 7.6,
      vote_count: 138,
      popularity: 32.969,
    },
    {
      adult: false,
      backdrop_path: "/uZTtVdOEIwPA6vwVRI3217DoPM.jpg",
      genre_ids: [35, 10749],
      id: 466282,
      original_language: "en",
      original_title: "To All the Boys I've Loved Before",
      overview:
        "Lara Jean's love life goes from imaginary to out of control when her secret letters to every boy she's ever fallen for are mysteriously mailed out.",
      poster_path: "/hKHZhUbIyUAjcSrqJThFGYIR6kI.jpg",
      release_date: "2018-08-17",
      title: "To All the Boys I've Loved Before",
      video: false,
      vote_average: 8.4,
      vote_count: 349,
      popularity: 31.76,
    },
    {
      adult: false,
      backdrop_path: "/yRXzrwLfB5tDTIA3lSU9S3N9RUK.jpg",
      genre_ids: [35, 18],
      id: 455980,
      original_language: "en",
      original_title: "Tag",
      overview:
        "For one month every year, five highly competitive friends hit the ground running in a no-holds-barred game of tag they’ve been playing since the first grade. This year, the game coincides with the wedding of their only undefeated player, which should finally make him an easy target. But he knows they’re coming...and he’s ready.",
      poster_path: "/eXXpuW2xaq5Aen9N5prFlARVIvr.jpg",
      release_date: "2018-06-14",
      title: "Tag",
      video: false,
      vote_average: 7,
      vote_count: 285,
      popularity: 87.194,
    },
    {
      backdrop_path: "/hHEqDPbO6z4Xje5tOf3Wm1mdMtI.jpg",
      first_air_date: "2018-08-17",
      genre_ids: [16, 35, 10765],
      id: 73021,
      name: "Disenchantment",
      origin_country: ["US"],
      original_language: "en",
      original_name: "Disenchantment",
      overview:
        "Set in a ruined medieval city called Dreamland, Disenchantment follows the grubby adventures of a hard-drinking princess, her feisty elf companion and her personal demon.",
      poster_path: "/c3cUb0b3qHlWaawbLRC9DSsJwEr.jpg",
      vote_average: 7.8,
      vote_count: 8,
      popularity: 19.929,
    },
  ],
  total_pages: 792,
  total_results: 15831,
};

const HomePage: React.FC = () => {
  const [openModal, setOpenModal] = useState(false);

  const handleClick = () => {
    // do nothing for now
  };

  const handleChange = () => {
    // do nothing for now
  };

  return (
    <ContainerComponent>
      <>
        <MoviesList items={moviesMock.results} />
        <div className="btn-wrapper">
          <ButtonComponent variant="round" onClick={handleClick}>
            Load
          </ButtonComponent>
        </div>
        <ButtonComponent variant="floating" onClick={() => setOpenModal(true)}>
          <RandomIcon />
        </ButtonComponent>

        <ModalComponent
          title="Movie Roulette"
          open={openModal}
          onClose={() => setOpenModal(false)}
        >
          <div className="mdl-container">
            <RadioGroupComponent
              groupName="Genre"
              items={genresMock.genres}
              title="Select genre:"
              onChange={handleChange}
            />
            <ButtonComponent variant="primary" onClick={handleClick}>
              Roll
            </ButtonComponent>
          </div>
        </ModalComponent>
      </>
    </ContainerComponent>
  );
};

export default HomePage;
