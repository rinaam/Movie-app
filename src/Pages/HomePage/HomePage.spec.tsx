import { act, create } from "react-test-renderer";
import HomePage from "./HomePage";
import React from "react";
import ButtonComponent from "../../Components/ButtonComponent/ButtonComponent";
import ModalComponent from "../../Components/ModalComponent/ModalComponent";

describe("HomePage", () => {
  it("should render HomePage", () => {
    const wrapper = create(<HomePage />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });

  it("should open modal", () => {
    const wrapper = create(<HomePage />);
    const openModalButton = wrapper.root.findAllByType(ButtonComponent)[1];

    act(() => {
      openModalButton.props.onClick();
    });

    const modalComponent = wrapper.root.findByType(ModalComponent);
    expect(modalComponent.props.open).toEqual(true);

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("should handle close modal", () => {
    const wrapper = create(<HomePage />);
    const openModalButton = wrapper.root.findAllByType(ButtonComponent)[1];

    act(() => {
      openModalButton.props.onClick();
    });

    const modalComponent = wrapper.root.findByType(ModalComponent);
    expect(modalComponent.props.open).toEqual(true);

    act(() => {
      modalComponent.props.onClose();
    });

    expect(modalComponent.props.open).toEqual(false);
  });
});
