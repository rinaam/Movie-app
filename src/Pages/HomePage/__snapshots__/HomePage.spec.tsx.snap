// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should open modal 1`] = `
<div
  className="wrapper"
>
  <div
    className="movie-list"
  >
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          8.3
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Avengers: Infinity War
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/to0spRl1CMDvyUbOnbb4fTk3VAd.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.6
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Deadpool 2
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/adOzdWS35KAo21r9R4BuFCkLer6.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.6
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Upgrade
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/hKHZhUbIyUAjcSrqJThFGYIR6kI.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          8.4
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            To All the Boys I've Loved Before
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/eXXpuW2xaq5Aen9N5prFlARVIvr.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Tag
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/c3cUb0b3qHlWaawbLRC9DSsJwEr.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.8
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Disenchantment
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
  </div>
  <div
    className="btn-wrapper"
  >
    <button
      className="btn btn-round"
      type="button"
    >
      Load
    </button>
  </div>
  <button
    className="btn btn-floating"
    type="button"
  >
    <IconMock />
  </button>
  <div
    className="modal"
  >
    <div
      className="container"
    >
      <div
        className="container-header"
      >
        <p>
          Movie Roulette
        </p>
        <button
          className="btn btn-close"
          type="button"
        >
          <IconMock />
        </button>
      </div>
      <div
        className="container-content"
      >
        <div
          className="mdl-container"
        >
          <div
            className="radio"
          >
            <p
              className="radio-title"
            >
              Select genre:
            </p>
            <label
              className="radio-option"
            >
              Action
              <input
                className="radio-option-btn"
                name="Genre"
                onChange={[Function]}
                type="radio"
                value="Action"
              />
            </label>
            <label
              className="radio-option"
            >
              Comedy
              <input
                className="radio-option-btn"
                name="Genre"
                onChange={[Function]}
                type="radio"
                value="Comedy"
              />
            </label>
            <label
              className="radio-option"
            >
              Drama
              <input
                className="radio-option-btn"
                name="Genre"
                onChange={[Function]}
                type="radio"
                value="Drama"
              />
            </label>
            <label
              className="radio-option"
            >
              SF
              <input
                className="radio-option-btn"
                name="Genre"
                onChange={[Function]}
                type="radio"
                value="SF"
              />
            </label>
          </div>
          <button
            className="btn btn-primary"
            type="button"
          >
            Roll
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`HomePage should render HomePage 1`] = `
<div
  className="wrapper"
>
  <div
    className="movie-list"
  >
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          8.3
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Avengers: Infinity War
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/to0spRl1CMDvyUbOnbb4fTk3VAd.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.6
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Deadpool 2
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/adOzdWS35KAo21r9R4BuFCkLer6.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.6
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Upgrade
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/hKHZhUbIyUAjcSrqJThFGYIR6kI.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          8.4
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            To All the Boys I've Loved Before
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/eXXpuW2xaq5Aen9N5prFlARVIvr.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Tag
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
    <div
      className="card"
      onClick={[Function]}
    >
      <div
        className="card-header"
      >
        <div
          className="card-header-image"
          style={
            Object {
              "backgroundImage": "url(\\"fake-base/w342/c3cUb0b3qHlWaawbLRC9DSsJwEr.jpg\\")",
            }
          }
        />
        <p
          className="card-header-grade"
        >
          7.8
        </p>
      </div>
      <div
        className="card-data"
      >
        <div
          className="card-data-title"
        >
          <p>
            Disenchantment
          </p>
          <p>
            (
            2018
            )
          </p>
        </div>
        <p>
           
          Language: en
        </p>
      </div>
    </div>
  </div>
  <div
    className="btn-wrapper"
  >
    <button
      className="btn btn-round"
      type="button"
    >
      Load
    </button>
  </div>
  <button
    className="btn btn-floating"
    type="button"
  >
    <IconMock />
  </button>
</div>
`;
